pages/Transactions/index.tsx

```diff
@@ -1,56 +1,174 @@
 import React, { useState } from 'react';
 import Layout from "../../components/Layout/Layout";
-import { Grid, Button, Tabs, Tab, Box } from "@mui/material";
+import { Button,
+   Tabs,
+   Tab,
+   Box,
+   List,
+   Input,
+   FormControl,
+   InputLabel,
+   Select,
+   MenuItem}
+from "@mui/material";
 import styles from './Transactions.module.css';
+import itemstyles from '../../components/TransactionItem/TransactionItem.module.css';
 import { TabPanel } from '../../components/TabPanel/TabPanel';
-
+import { TransactionItem } from '../../components/TransactionItem/TransactionItem';
+import { SortTriangles } from '../../components/SortTriangles/SortTriangles';
+import Icon from '../../assets/Icon';
+ 
 const TransactionsPage: React.FunctionComponent = () => {
-
-    const BasicTabs = () => {
-        const [value, setValue] = React.useState(0);
-        
-        const handleChange = (event, newValue) => {
-            setValue(newValue);
-        };
-        
-        return (
-            <Box sx={{ width: '100%', maxHeight: '62vh' }}>
-            <Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
-                <Tabs value={value} onChange={handleChange} aria-label="basic tabs example">
-                <Tab label="History" className={value==0 ? styles['seltab'] : styles['tab']}/>
-                <Tab label="Redemptions" className={value==1 ? styles['seltab'] : styles['tab']}/>
-                <Tab label="Earnings" className={value==2 ? styles['seltab'] : styles['tab']}/>
-                </Tabs>
-            </Box>
-                <TabPanel value={value} index={0}>
-                    Item One
-                </TabPanel>
-                <TabPanel value={value} index={1}>
-                    Item Two
-                </TabPanel>
-                <TabPanel value={value} index={2}>
-                    Item Three
-                </TabPanel>
-            </Box>
-        );
-    }
+ 
+   const BasicTabs = () => {
+       const [value, setValue] = React.useState(0);
```

Comment:
nit: since we already imported the useState hook, just use it directly here
```suggestion
       const [value, setValue] = useState(0);
```